src/handlers/HandlerFactory.ts

```diff
@@ -0,0 +1,6 @@
+import StakeRequestedHandler from './StakeRequestedHandler';
+
+
+export default {
+  stakeRequesteds: new StakeRequestedHandler(),
```

Comment:
This can be merged with Facilitator.getSubscriptionDetails() method.
We have similar information there. We can merge below to a single class/method.
So for stakeRequested entity => Subscription query => StakeRequestedHandler.
Object can look like below:
```
{
  stakeRequested: { subscriptionQuery: "", handler: StakeRequestedHandler}
}
```
You can decide on the right placeholder 🚮

Reply:
If you see latest implementation of HandlerFactory, it's way more complex. This now has responsibility of instantiating all the handler which would require instantiation of repositories and services.

Reply:
I still think we should merge entity wise handler, subscription queries together  to have single placeholder. Easy to access and read
There will be some refactoring needed in Facilitator.getSubscriptionDetails. We can take up at that time.